lib/simde/simde/x86/sse.h

```diff
@@ -31,9 +31,11 @@
 #define SIMDE_X86_SSE_H
 
 #include "mmx.h"
+#include "../simde-f16.h"
 
 #if defined(_WIN32) && !defined(SIMDE_X86_SSE_NATIVE) && defined(_MSC_VER)
-  //#include <windows.h>
+  #define NOMINMAX
+  #include <windows.h>
```

Comment:
Shall I comment out this line again?

Reply:
thanks, Boxedwine goes out of its way not to include windows.h except for in a couple of specific files.